### DISCUSSION TITLE
No browsers after clean 15.1 install

### DISCUSSION THREAD

**Discussion by Tachi107** [Divested-Mobile/DivestOS-Build#302] [Q&A]
Hi all! After performing a clean install of the latest DivestOS 15.1 release (15.1-20240408) on my Nexus 5X (bullhead), I've noticed that there are no browsers installed out of the box. Not only that, but it also seems that there's no certificate store on the device; after installing an application I'm working on via adb, it failed to connect to the internet and logcat was showing issues related to TLS certificate validation.

I had a look at the list of known issues, but I saw no mention of this. Am I doing something wrong?

Thanks!

**Comment by SkewedZeppelin** [comment]
https://divestos.org/pages/broken#papercuts
> Mull is not installed by default [due to size constraints], only an empty shim/placeholder is. F-Droid will prompt you to "update" it on first run which will install it.

There is very much a certificate store included and my devices (bullhead, flounder, and dragon) under 15.1 work fine.

Was your time correct?

edit: just wiped my bullhead, all certs were as expected, but time was set to 1971

**Reply by Tachi107** [reply]
Hi, thanks for your reply. How dumb of me, the failing certificate validation must've been caused by the wrong date, thanks for pointing it out!

After wiping /data and inserting a SIM card to have Internet access during initial setup (Wi-Fi and Bluetooth have been broken for years on that phone, after handing it to a repair shop...), everything works as expected! Sorry for the useless question :)
